### ISSUE TITLE
Is it possible to run that in the emulator ?

### ISSUE THREAD

**Issue by AlexPiro** [sceneunderstanding-microsoft/unitysample#10]
I try to use the scene understanding in a project. It's working fine in the Unity Editor but nothing happens in the HoloLens 2 emulator when deployed. I only see the spatial meshes but no colors or labels of the Scene Understanding functionalities.
So it is probably not working in the emulator. Has anyone already tried that ?

**Comment by SzymonSPS** [comment]
Hi Alex, can you confirm if you're talking about the emulator or the remote renderer? (https://docs.microsoft.com/en-us/windows/mixed-reality/using-the-hololens-emulator)  Scene Understanding does work in the remote renderer but not in the emulator, as we have not implemented some of the hardware to do scene understanding processing.

It might be possible to us the emulator and pre-generated scenes. If you can give a little more info on what you want to do we may be able to suggest some approaches.

**Comment by AlexPiro** [comment]
Yes I tried to use this sample in the HoloLens 2 emulator (not the remote renderer).
`SceneObserver.IsSupported()` return `false`
As I didn't received my HoloLens 2 yet, I'm still working with the emulator to ensure the app works outside of the Unity Editor. But if we are using the scene understanding or making an experience with both HL1 and HL2, we are stuck.

**Comment by SzymonSPS** [comment]
Right, you still would need to run the "PC" path by disabling the "Run on device" box even in the emulator so that the SceneObserver is not used, and instead the scene is deserialized from disk. SceneObserver is the object that communicates with the device itself and it won't work under emulation, but once you have a scene then everything is available to you.
